mlflow/server/js/src/components/ExperimentView.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { ExperimentView } from './ExperimentView';
+import KeyFilter from "../utils/KeyFilter";
+import {LIFECYCLE_FILTER} from "./ExperimentPage";
+import Fixtures from '../test-utils/Fixtures';
+
+const getExperimentViewMock = () => {
+  return shallow(<ExperimentView
+    onSearch={() => {}}
+    runInfos={[]}
+    experiment={Fixtures.experiments[0]}
+    history={[]}
+    paramKeyList={[]}
+    metricKeyList={[]}
+    paramsList={[]}
+    metricsList={[]}
+    tagsList={[]}
+    paramKeyFilter={new KeyFilter("")}
+    metricKeyFilter={new KeyFilter("")}
+    lifecycleFilter={LIFECYCLE_FILTER.ACTIVE}
+    searchInput={""}
+  />);
+};
+
+test('Entering filter input updates component state', () => {
+  const wrapper = getExperimentViewMock();
+  wrapper.instance().setState = jest.fn();
```

Comment:
Can you validate stuff with `wrapper.state`?

Reply:
Good idea - I tried, but end up getting the same error described in https://github.com/airbnb/enzyme/issues/1570.